example/ios/SDWebImageDownloaderOperation+Swizzle.m

```diff
@@ -0,0 +1,102 @@
+//
+//  SDWebImageDownloaderOperation+Swizzle.m
+//  NetworkClientExample
+//
+//  Created by Miguel Alatzar on 12/2/20.
+//  Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+//  See LICENSE.txt for license information.
+//
+
+#import "SDWebImageDownloaderOperation+Swizzle.h"
```

Comment:
do we want to link this file in the README file for an example on how to use Swizzling with this lib?